sos/report/plugins/ovn_central.py

```diff
@@ -82,29 +88,41 @@ def setup(self):
             self.add_copy_spec("/var/log/ovn/*.log")
 
         # Some user-friendly versions of DB output
-        cmds = [
+        nbctl_cmds = [
             'ovn-nbctl show',
-            'ovn-sbctl show',
-            'ovn-sbctl lflow-list',
             'ovn-nbctl get-ssl',
             'ovn-nbctl get-connection',
-            'ovn-sbctl get-ssl',
-            'ovn-sbctl get-connection',
             'ovn-nbctl list loadbalancer',
             'ovn-nbctl list Load_Balancer',
             'ovn-nbctl list ACL',
             'ovn-nbctl list Logical_Switch_Port',
         ]
 
+        sbctl_cmds = [
+            'ovn-sbctl show',
+            'ovn-sbctl lflow-list',
+            'ovn-sbctl get-ssl',
+            'ovn-sbctl get-connection',
+        ]
+
         schema_dir = '/usr/share/openvswitch'
 
         nb_tables = self.get_tables_from_schema(os.path.join(
             schema_dir, 'ovn-nb.ovsschema'))
-        sb_tables = self.get_tables_from_schema(os.path.join(
-            schema_dir, 'ovn-sb.ovsschema'), ['Logical_Flow'])
 
-        self.add_database_output(nb_tables, cmds, 'ovn-nbctl')
-        self.add_database_output(sb_tables, cmds, 'ovn-sbctl')
+        self.add_database_output(nb_tables, nbctl_cmds, 'ovn-nbctl')
```

Comment:
Since `nbctl_cmds` is a smaller set than original `cmds`, we do collect here less database outputs. Is that intentional change? (I *think* so, as nb_tables ~ nbctl only, but I know nothing about OVN).

An equivalent comment applies to `self.add_database_output(sb_tables, sbctl_cmds, 'ovn-sbctl')` on line 124.

Reply:
yes if we cant access the sbdb we cant get that info